requirements.txt

```diff
@@ -7,7 +7,7 @@ charset-normalizer==2.0.12
 click==8.1.3
 craft-cli==0.6.0
 craft-grammar==1.1.1
-craft-parts==1.7.1
+craft-parts @ git+https://github.com/mr-cal/craft-parts.git@architectures
```

Comment:
I would like to get @cmatsuoka opinion on how I am passing the architecture into and out of `craft-parts`.  Currently `craft-parts` is not happy about parsing `arch=all`.

https://github.com/canonical/craft-parts/compare/main...mr-cal:architectures